_data/success-stories.yml

```diff
@@ -0,0 +1,80 @@
+cases:
+  - name: Netflix
+    link: https://medium.com/netflix-techblog/global-continuous-delivery-with-spinnaker-2a6896c23ba7
+    logo: cases-logos/netflix.jpg
+    text: “The community uptake has been tremendous! Having a platform that out-of-the-box accommodates multiple cloud providers and an extendable model to participate within, means that we are one large community rather than a bunch of micro-communities focused on various forks. This has the benefit that everyone within the community can take advantage of various innovations.”
```

Comment:
@brian-armory 
@fifthposition 
Do we want to correct respondents' words? In this case, s/extendable/extensible/.

Reply:
I'm always weary of editing people's actual words in cases like this, so I'll defer to you and @fifthposition

Reply:
In the linked blog post, the word "extendable" does not appear. "Extensible" does appear, but in a different context.